packages/commonwealth/server/controllers/server_communities_methods/delete_community.ts

```diff
@@ -198,8 +117,46 @@ export async function __deleteCommunity(
             },
           );
 
-          await this.models.Address.destroy({
-            where: { community_id: community.id },
+          const models = [
+            this.models.CommunitySnapshotSpaces,
+            this.models.CommunityStake,
+            this.models.DiscordBotConfig,
+            this.models.Ban,
+            this.models.Reaction,
+            this.models.Comment,
+            this.models.Topic,
+            this.models.Subscription,
+            this.models.CommunityContract,
+            this.models.Webhook,
+            this.models.Vote,
+            this.models.Poll,
+            this.models.Thread,
+            this.models.StarredCommunity,
+            this.models.CommunityBanner,
+            this.models.Notification,
+            this.models.Group,
+            this.models.Address,
+          ];
+
+          for (const model of models) {
+            await (model as ModelStatic<any>).destroy({
```

Comment:
We can actually do better than `any` here, I bet, by adding the community_id field at least (so we can ensure provided models meet the type specification. We should also move the type declaration up to the variable declaration, so it's applied at assignment rather than on iteration.